### PR TITLE
Updating new address in the website

### PR DESCRIPTION
**Pull request by AnneThKu**
Below updates are done in the website:
- Updated new address and directions in Contact page.
- Replaced the SWAN Brochure and SWAN White paper with the ones having the new address.
- Replaced Gen4 modem brochure, Multi-channel modem brochure and WNC White paper for new address changes.
- Updated the SWAN Brochure in the QuickLinks page.

### CODE REVIEW

**Comment by notthetup** [comment]
While we're at it, can we move all the static content to data files?

File: brochures/index.html [hunk 1]
```diff
@@ -16,12 +16,8 @@ <h1> Brochures </h1>
 		<a href="{{site.baseurl}}/brochures/Subnero-Multi-Channel-Modems.pdf" target="_blank" style="font-size: 1.2em;">Subnero Multi-channel Underwater Modems</a>
 	</div>
 	<div class="brochure-container">
-		<a href="{{site.baseurl}}/brochures/subnero-swan.pdf"><img class="brochure-thumb" src="swan.jpg"></a>
-		<a  href="{{site.baseurl}}/brochures/subnero-swan.pdf" target="_blank" style="font-size: 1.2em;">Subnero Water Assesment Network (SWAN)</a>
-	</div>
-	<div class="brochure-container">
-	  <a href="{{site.baseurl}}/brochures/SWAN_Brochure_Chinese_v3.1.pdf"><img class="brochure-thumb" src="{{site.baseurl}}/brochures/swan.jpg"></a>
-	  <a  href="{{site.baseurl}}/brochures/SWAN_Brochure_Chinese_v3.1.pdf" target="_blank" style="font-size: 1.2em;">Subnero Water Assessment Network (SWAN) (Mandarin)</a>
+		<a href="{{site.baseurl}}/brochures/SWAN_Brochure.pdf"><img class="brochure-thumb" src="swan_brochure.jpg"></a>
```
Comment (manuignatius) [inline brochures/index.html:19]: If the thumbnail image is new, can you delete the old one?

**Review by manuignatius — DISMISSED** [review]
I have a comment, otherwise looks good.

File: brochures/index.html (hunk 1 above)
Comment (AnneThKu) [reply brochures/index.html:19]: Removed the old SWAN brochure and thumbnail.

**Comment by AnneThKu** [comment]
I have moved the static content in the contacts page as front matter variables (https://github.com/subnero1/subnero1.github.io/pull/214/commits/398e60cdab52ae22e0aeb33ec9c2beef39e8a64a).

**Review by notthetup — APPROVED** [review]

**Pull request merged**